lib/resource.js

```diff
@@ -63,22 +62,16 @@ export default class PodiumClientResource {
 
         this.#state.setInitializingState();
 
-        const chunks = [];
-        const to = new stream.Writable({
-            write: (chunk, encoding, next) => {
-                chunks.push(chunk);
-                next();
-            },
-        });
-
-        stream.pipeline([outgoing, to], () => {
-            // noop
-        });
-
         const { manifest, headers, redirect } = await this.#resolver.resolve(
             outgoing,
         );
 
+        const chunks = [];
+        // eslint-disable-next-line no-restricted-syntax
+        for await (const chunk of outgoing) {
+            chunks.push(chunk)
+        }
```

Comment:
I was unable to get pipeline to work here but Undici bodies support for await of which worked fine

Reply:
:+1: